app/react/App/scss/modules/_document.scss

```diff
@@ -295,6 +295,7 @@ mark.searchTerm {
 
 .copy-from {
   flex: 1;
+  min-width: 50%;
```

Comment:
I'm not sure I understand the reasoning behind this combo.  Could the copy-from pane be bigger than 50%?  If not, this should not be a `flex: 1` with `min-width`, it should be a simpler `width: 50%`.  Maybe this is for responsive behavior?

Reply:
it can be in mobile